### ISSUE TITLE
Add send-envelope command 

### ISSUE THREAD

**Issue by smeubank** [getsentry/sentry-cli#1168]
Add send-envelope command ([blocked by missing features in sentry-rust](https://github.com/getsentry/sentry-rust/issues/381))

    - given some file, send evelope with DSN - rust cannot create an envelope

**Comment by kamilogorek** [comment]
Duplicate of https://github.com/getsentry/sentry-cli/issues/703
